README.md

```diff
@@ -28,7 +31,7 @@ This is a use case diagram. Use case diagrams overview the usage requirements fo
 
 ## Requirements
 
-In order to use this, the user must be IOHK Zendesk agent and be able to provide email and password to the classifier.
+In order to use this, the user must be possess an IOHK Zendesk agent ID and API key.
```

Comment:
There is a "be" left over. A correct way to write it would be: "In order to use this, the user must possess an IOHK Zendesk agent ID and API key."

Reply:
Good catch, thank you.